File: tests/code/corpus/langs/swift/Registry.swift
import Foundation

class Registry {
    private var items: [Any] = []

    func add(item: Any) {
        items.append(item)
    }

    func seed() {
        add(item: makeCircle(radius: 1.0))
    }
}

func emptyRegistry() -> Registry {
    return Registry()
}

File: tests/code/corpus/langs/php/Registry.php
<?php

namespace App\Geometry;

class Registry
{
    private $items = [];

    public function add($item)
    {
        $this->items[] = $item;
    }

    public function seed()
    {
        $this->add(makeCircle(1.0));
    }
}

function emptyRegistry()
{
    return new Registry();
}
